client/src/Components/SideNav.js

```diff
@@ -21,6 +22,7 @@ class SideNav extends Component {
         <li><Link to='/compare_players'>Compare Players</Link></li>
         <li><Link to='/team_dashboard'>Team Dashboard</Link></li>
         <li><Link to='/trade_simulator'>Trade Simulator</Link></li>
+        <li><Link to='/VolunteerUrl'>Volunteer</Link></li>
```

Comment:
This currently links to a webpage at /VolunteerUrl which doesn't exist. The line of code that you want is down below:

```
<li><a href={volunteerUrl} target='_blank'> Volunteer</a></li>
```

The squiggly braces substitute the variable into the HTML while the other is just a plain string 😺

Reply:
Noted.